src/js/components/Tabs/stories/CustomThemed/AlignControls.js

```diff
@@ -51,8 +51,8 @@ const AlignControlsTabs = () => (
 );
 
 export const AlignControls = () => <AlignControlsTabs />;
-AlignControls.storyName = 'Align controls';
+AlignControls.storyName = 'Align Controls';
```

Comment:
```suggestion
AlignControls.storyName = 'Align controls';
```
Folders in storybook should use title case and individual stories should use sentence case